packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -507,6 +507,13 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
         'SocketException occurred, could not reach Gold. '
           'Switching to FlutterSkippingGoldenFileComparator.',
       );
+    } on FormatException catch (_) {
```

Comment:
I've not seen this error before from Gold in this context.

@kjlubick have there been any changes to `https://flutter-gold.skia.org/json/v2/latestpositivedigest/$traceID`?

Reply:
No changes. A socket exception seems like something in the network stack/path. :shrug:

Reply:
Ok thanks. I was referring to the FormatException, but it sounds like everything is stable! :)

Reply:
Maybe a reply got truncated? Thus JSON parsing failed?